apps/aws-notebook/samples/ecs/fargate/vpn/README.md

```diff
@@ -0,0 +1,38 @@
+# Scripts to quicky deploy a aws-notebook  development environment
+This will create a AWS ECS cluster and a aws-notebook sandbox app by some commands.
+
+The only thing you need to prepare is a image for your ecs service. You should build it from this [dockerfile](https://github.com/crafting-demo/solutions/blob/master/apps/aws-notebook/image/Dockerfile) and push it to your aws container registry
+
+
+## Usage
+```bash
+# Create AWS ECS
+terraform init && terraform apply -auto-approve 
+
+# Create a sandbox app for aws-notebook
+## Export AWS config
+export AWS_REGION=xxxx
+export AWS_ACCESS_KEY_ID=xxxxx
+export AWS_SECRET_ACCESS_KEY=xxxxxx
+export AWS_SESSION_TOKEN=xxxxx
```

Comment:
Using AWS credentials is not recommended in sandbox env. We highly recommend using Identity federation. The content of the config file is like:

```
[default]
region = <REGION>
credential_process = idfed aws <ACCOUNT-ID> <ROLE>
```